pydm/widgets/timeplot.py

```diff
@@ -369,6 +379,9 @@ class PyDMTimePlot(BasePlot):
     SynchronousMode = 1
     AsynchronousMode = 2
```

Comment:
We can probably clean these up in some way if the user never saw them previously. The previous interface had the user giving a bool rather than an enum.

Reply:
Updated variables AsynchronousMode  and SynchronousMode to _AtFixedRate_ and _OnValueChange_ for naming consistency